tests/decision/test_decision.py

```diff
@@ -153,15 +156,31 @@ def test_generator(self):
 class TestDecisionManager():
 
     def test_null_strategy(self):
-        df = DecisionManager([2010, 2015], [], [])
+        handle = Mock(timesteps=(2010, 2015))
+        handle.interventions = []
+        store = MemoryInterface()
+        store._model_runs = {'test': {'sos_model': 'test_sos_model'}}
+        store._sos_models = {'test_sos_model': {'sector_models': []}}
+        store._strategies = {'test': []}
+        handle._store = store
+        handle._modelrun_name = 'test'
+        df = DecisionManager(handle)
         dm = df.decision_loop()
         bundle = next(dm)
         assert bundle == {0: [2010, 2015]}
         with raises(StopIteration):
             next(dm)
 
-    def test_decision_manager_init(self, get_strategies):
-        df = DecisionManager([2010, 2015], get_strategies, [])
+    def test_decision_manager_init(self):
```

Comment:
This seems to duplicate `test_null_strategy` above.

Reply:
Duplicate test removed